src/brs/Generator.java

```diff
@@ -41,6 +41,8 @@
 
     BigInteger getDeadline();
 
+    BigInteger getDeadlineLegacy();
```

Comment:
Legacy deadline (without the new formula so miners can be oblivious to the deadline change)